src/components/TableHeader/TableHeader.tsx

```diff
@@ -48,6 +50,7 @@ export default function TableHeader({
         className={columnClassNames[columnIndex]}
         ariaColIndex={ariaColIndex}
         ariaRowIndex={ariaRowIndex}
+        columnConfig={columnConfig}
```

Comment:
maybe `columnDescriptor` as it's a `ColumnDescriptor`, not only a `ColumnConfig` instance